pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ControllerFilePathProvider.java

```diff
@@ -126,14 +126,17 @@ public URI getDataDirURI() {
   }
 
   public File getFileUploadTempDir() {
+    org.apache.pinot.common.utils.FileUtils.createDirIfNotExists(_fileUploadTempDir.toPath());
```

Comment:
please import `FileUtils` itself

Reply:
`apache.commons.io.FileUtils` is already present

Reply:
Oh sorry my bad!